src/System.Drawing.Common/src/System/Drawing/Design/CategoryNameCollection.cs

```diff
@@ -3,68 +3,23 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections;
+using System.Security.Permissions;
 
 namespace System.Drawing.Design
 {
-    /// <summary>
```

Comment:
I'd rather keep them. We don't actually maintain any other doc comments, and ideally we will start producing and shipping the documentation from these source files. We do ship the comments from some libraries, like System.Numerics.Vectors. Other libraries don't even ship XML docs even though the source code has them 😞 .